cost/scheduled_reports/scheduled_report.pt

```diff
@@ -20,7 +20,7 @@ category "Cost"
 default_frequency "monthly"
 tenancy "single"
 info(
-  version: "1.9",
+  version: "2.0",
```

Comment:
version should be 1.20.

Reply:
I have addressed the comment please review the changes.